src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,83 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Represents a <code>Storage</code> object to handle saving and loading.
+ */
+
+
+public class Storage {
+    private String filepath;
+    public Storage(String filepath) {
+        this.filepath = filepath;
+    }
+
+    /**
+     * Saves current tasklist as a file in specified filepath.
+     * @param tasklist
+     */
+    public void save(TaskList tasklist) {
+        try {
+            List<Task> ls = tasklist.getList();
+            File f = new File(filepath);
+            f.delete();
+            f.createNewFile();
+            FileWriter fw = new FileWriter(filepath);
+            for (Task t : ls) {
+                fw.write(t.saveString() + "\n");
+            }
+            fw.close();
```

Comment:
I suggest closing `File` and `FileWriter` in finally block. This [link](https://stackoverflow.com/questions/37037718/is-it-necessary-to-call-close-in-a-finally-when-writing-files-in-java) should help:D I noticed similar issue in other places as well.